src/platforms/apple/common/performance/instrumentation/automatic-instrumentation.mdx

```diff
@@ -0,0 +1,80 @@
+
+Capturing transactions requires that you first <PlatformLink to="/performance/">set up performance monitoring</PlatformLink> if you haven't already.
+
+</Note>
+
+## UIViewController Instrumentation
+
+This feature is available for iOS, tvOS, and Mac Catalyst, and only works for UIViewControllers.
+
+The UIViewController Instrumentation, once enabled, captures transactions whenever your app loads a ViewController. The SDK sets the transaction name to the name of the ViewController, including the module; for example, `Your_App.MainViewController`, and the transaction operation to `ui.load`. The SDK creates spans to provide insight into the length of time each consumed by each of the following methods.
+
+![UIViewController Transaction](ui-view-controller-transaction.png)
+
+You can opt out of this feature using options:
+
+```swift {tabTitle:Swift}
+import Sentry
+
+SentrySDK.start { options in
+    options.dsn = "___PUBLIC_DSN___"
+    options.enableAutoUIPerformanceTracking = false
+}
+```
+
+```objc {tabTitle:Objective-C}
+@import Sentry;
+
+[SentrySDK startWithConfigureOptions:^(SentryOptions *options) {
+    options.dsn = @"___PUBLIC_DSN___";
+    options.enableAutoUIPerformanceTracking = NO;
+}];
+```
+
+## App Start Instrumentation
+
+This feature is available for iOS, tvOS, and Mac Catalyst.
+
+The App Start Instrumentation provides insight into how long your application takes to launch. It adds spans for different phases, from the application launch to the first auto-generated UI transaction.
+To enable this feature enable `AutoUIPerformanceTracking` and set a <PlatformLink to="/configuration/sampling/#setting-a-uniform-sample-rate">`tracesSampleRate`</PlatformLink> or a <PlatformLink to="/configuration/sampling/#setting-a-sampling-function">`tracesSampler`</PlatformLink>. The SDK differentiates between a cold and a warm start, and doesn't track hot starts/resumes.
+* __Cold start__: App launched for the first time, after a reboot or update. The app is not in memory and no process exists.
+* __Warm start__: App launched at least once, is partially in memory, and no process exists.
+
+The SDK uses the process start time as the beginning of the app start and the [`UIWindowDidBecomeVisibleNotification`][UIWindow] as the end. It creates the following spans:
+* __Pre Main__: From the process start time to the runtime init.
+* __UIKit and Application Init__: From the runtime init to the [`didFinishLaunchingNotification`][didFinishLaunching].
+* __Initial Frame Render__: From the [`didFinishLaunchingNotification`][didFinishLaunching] to [`UIWindowDidBecomeVisibleNotification`][UIWindow].
+
+![App Start Transaction](app-start-transaction.png)
+
+You can opt out of this feature using options:
+
+```swift {tabTitle:Swift}
+import Sentry
+
+SentrySDK.start { options in
+    options.dsn = "___PUBLIC_DSN___"
+    options.enableAppStartMeasuring = false
```

Comment:
I thought that we decided not offering such an option as it's bounded to the `enableAutoUIPerformanceTracking` anyway.

Reply:
Okay, then I'm going to remove it here and create an issue for removing it in Cocoa. We can always add such a feature later on.